server/githookHandler.go

```diff
@@ -0,0 +1,84 @@
+package server
+
+import (
+	"bytes"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/eventgrid/2018-01-01/eventgrid"
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/go-autorest/autorest/date"
+	"github.com/Azure/go-autorest/autorest/to"
+	"github.com/gobuffalo/buffalo"
+	"github.com/gobuffalo/buffalo/render"
+	"github.com/google/go-github/github"
+	"github.com/google/uuid"
+)
+
+// GithookReceiver converts a Github event into an Eventgrid event and publishes the event to Eventgrid
+func GithookReceiver(c buffalo.Context) error {
+	webhookSecret := os.Getenv("webhookSecret")
+	eventgridSecret := os.Getenv("eventgridSecret")
+	eventgridEndpoint := os.Getenv("eventgridEndpoint")
```

Comment:
When reading configuration like this in a Buffalo application, it's more typical to use [`envy`](https://github.com/gobuffalo/envy) than the Standard Library directly. It should automatically amend the environment you're executing in with vars that are defined in a `.env` file local to the project.

For my personal stuff that aren't buffalo applications, I prefer [`viper`](https://github.com/spf13/viper).

Reply:
is there a better way to inject them into the handler as well? seems funny to grab them each time the handler is called

Reply:
You could have global-variables that hold them, and are populated exactly once in `func init()`. However, this would mean that any updates to the environment wouldn't get picked up until the process restarted. I'm not sure about `envy`, but I know dynamic environment is supported by `viper`.